April CQA work for MTA 8.1 Release Notes DITA fixes (#356)
migtools/mta-documentation: assemblies/release-notes/assembly_mta-8-0.adoc

:context: mta-8-0

[role="_abstract"]

> Review the Release Notes for new features, enhancements, and fixes in {ProductFullName} 8.0.

Reviewer: ```suggestion
Review new features, enhancements, and fixed issues in {ProductFullName} 8.0.
```
